cmd/daemon/init.go

```diff
@@ -53,15 +59,12 @@ func buildInitCmd(parentCmd *cobra.Command) {
 			err := wallet.CheckMnemonic(*restoreOpt)
 			cmd.FatalErrorCheck(err)
 		}
-		cmd.PrintLine()
-		cmd.PrintInfoMsgf("Enter a password for wallet")
-		password := cmd.PromptPassword("Password", true)
 
-		cmd.PrintLine()
-		cmd.PrintInfoMsgBoldf("How many validators do you want to create?")
-		cmd.PrintInfoMsgf("Each node can run up to 32 validators, and each validator can hold up to 1000 staked coins.")
-		cmd.PrintInfoMsgf("You can define validators based on the amount of coins you want to stake.")
-		numValidators := cmd.PromptInputWithRange("Number of Validators", 7, 1, 32)
```

Comment:
Still we break the previous behaviour. If user didn't set the number of validator, it should work as before.

Reply:
It should be like:
```
var valNum int
if *valNumOpt == 0 {
// .... old code
} else {
// ... new code
}
```